test/Internal.AspNetCore.Analyzers.Tests/PubternabilityAnalyzerTests.cs

```diff
@@ -93,6 +93,20 @@ public class T
             Assert.Empty(diagnostics);
         }
 
+
+        [Theory]
+        [MemberData(nameof(PublicTypeWithAllowedDefinitions))]
+        public async Task PublicExposureOfPubternalTypeSometimesAllowed(string member)
```

Comment:
`PublicExposureOfPubternalType_AllowedForTypesImplementingPubInternalInterfaces`. Would help to be specific

Reply:
Not sure that interfaces would be the only case in the future.